scripts/move_to_spark_1.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set +x
+
+grep -q "spark2" pom.xml
```

Comment:
Yeah, I agree that it is non-obvious though. Let me touch it up with some inline docs to draw attention to what's going on here.